src/portal/views/Dao.vue

```diff
@@ -373,7 +378,9 @@ export default class DaoView extends Vue {
                       { title: "Archived", content: proposals.inactive },
                     ];
                   }
-                );
+                ).finally(() => {
+                  this.loadingProposals = false;
+                });
```

Comment:
I don't see why you'd include this here as the loading proposals isn't displayed when you are voting